composeApp/src/commonMain/kotlin/sp/bvantur/tasky/event/ui/EventDetailsScreen.kt

```diff
@@ -101,10 +106,10 @@ fun CreateEventRoute(
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun CreateEventScreen(
+fun EventDetailsScreen(
```

Comment:
Not sure if one of use already shared this with you but, one of the tougher decisions to make for this project is whether to use one screen per agenda item type or one screen for all agenda items.

A concern was that having a single screen makes it harder to read and update, but it actually got a lot easier:
If you look at the UI hierarchy of the items, reminder and task screens are almost identical while the event screen also has the same elements with a few extra composables. 

The complete root screen structure is no more than this:
```
HeaderSection()
TitleSection()
DescriptionSection()
if(item is Event) {
    PhotoSection()
}
StartDateSection()
if(item is Event) {
    EndDateSection()
}
ReminderSection()
if(item is Event) {
    AttendeeSection()
}
DeleteButton()
```
As long as you extract the UI into single components, it remains very readable. Same goes for the underlying ViewModel, you will save so much code by just having one ViewModel, since the logic of what all items have in common only needs to be written once. And if you'd extend that screen with another type of agenda item, that would also be easier and wouldn't require you to duplicate most of this to another screen.

Reply:
I am planning to do that when I will get to the second type for this screen. I will rename files and then try to replicate a scenario at work for myself when the first requirement is only for Events. Then, I get a new task that requires supporting a new type of agenda item.